taxi/tests/test_forms.py

```diff
@@ -0,0 +1,18 @@
+from django.test import TestCase
+
+from taxi.forms import DriverCreationForm
+
+
+class MyTestCase(TestCase):
+    def test_driver_creation_form(self):
+        form_data = {
+            "username": "new_driver",
+            "password1": "user1test",
+            "password2": "user1test",
+            "first_name": "new",
+            "last_name": "driver",
+            "license_number": "ASD12345"
+        }
+        form = DriverCreationForm(data=form_data)
+        self.assertTrue(form.is_valid())
+        self.assertEqual(form.cleaned_data, form_data)
```

Comment:
The `assertEqual` statement may not work as expected because `cleaned_data` does not include password fields in the same format as `form_data`. Consider checking individual fields or excluding password fields from the comparison.